op-guide/requirement.md

```diff
@@ -0,0 +1,77 @@
+| PD      | 16 core+   | 16 GB+  | SAS, 200 GB+ | Gigabit network card | -       |
+| TiKV    | 16 core+   | 32 GB+  | SAS, 200 GB+ | Gigabit network card | 3       |
+|         |         |         |              | Total Server Number |  4      |     
+
+**Note**
+
+- Verify that in the test environment the TiDB and PD can be deployed on the same server.
+- In performance-related testing, do not use low-performance storage and network hardware configuration, in order to guarantee the correctness of the test result.
+
+### Production Environment
+
+| Component | CPU | Memory | Hard Disk Type | Hard Disk Number | Single Hard Disk Size | Network | Instance Number (lowest requirement) |
+| :-----: | :------: | :------: | :------: | :------: | ------ | :------: | :-----: |
+|  TiDB  | 32 core+ | 128 GB+ | SSD | minimum 2 | 500 GB+ | 2+ 10 Gigabit network card | 2 |
+| PD | 16 core+ | 32 GB+ | SSD | minimum 2 | 200 GB+ | 2+ 10 Gigabit network card | 3 |
+| TiKV | 32 core+ | 128 GB+ | SSD | minimum 2 | 200~500 GB. It is recommended to keep the size of TiKV hard disk within 500G in case it takes too long to restore data when the hard disk is damaged.  | 2+ 10 Gigabit network card | 3 | 
+| Monitor | 16 core+ | 32 GB+ | SAS | minimum 4 | 200 GB+ | 2+ 10 Gigabit network card | 1 |
+|     |     |     |      |      |      |  Total Server Number   |    9   |
+
+**Note**:   
+
+- In the production environment, you can deploy and operate TiDB and PD on the same server. If you have a higher requirement for performance and reliability, try to deploy them separately.
+- It is strongly recommended to use higher configuration in the production environment.
+
+## Network Requirements
+
+As an open source distributed NewSQL database, TiDB requires the following network port configuration to operate. Based on the TiDB deployment in actual environments, the administrator can enable relevant ports in the network side and host side. 
+
+| Component | Default Port | Description |
+| :--:| :--: | :-- |
+| TiDB |  4000  | access communication port of application and DBA tools|
+| TiDB | 10080  | communication port to report TiDB status|
+| TiKV |  20160 |  TiKV communication port  |
+| PD | 2379 | to provide TiDB and PD communication port|
+| PD | 2380 | PD cluster inter-node communication port|
+| Prometheus |  9090|Prometheus service communication port|
+| Pushgateway |  9091| TiDB, TiKV, PD monitor aggregation and report port|
+| Node_exporter |  9100| communication port to report the system information of every TiDB cluster node |
+| Grafana | 3000 | Web monitor external service and client (browser) access port|
+
+## Client Web Browser Requirements
+
+Based on the Prometheus and Grafana platform, TiDB provides a visual data monitoring solution for distributed database clusters. To visit the Grafana monitor entry, it is recommended to use a higher version of Microsoft IE, Google Chrome or Mozilla Firefox.
```

Comment:
Based on the Prometheus and Grafana platform, TiDB provides a visual data monitoring solution to monitor the TiDB cluster status. To visit the Grafana monitor interface, it is recommended to use a higher version of Microsoft IE, Google Chrome or Mozilla Firefox.